source/security/private-endpoints.txt

```diff
@@ -67,6 +71,11 @@ You can add a new Private Endpoint with the App Services UI or the Admin API.
 You can add a new General Endpoint or a new Sync Endpoint. You can also
 migrate a Legacy Endpoint to either a new General or Sync Endpoint.
 
+.. note:: 
+   
+   Endpoints that use the ``realm.mongodb.com`` domain must migrate to use the new
+   ``services.cloud.mongodb.com`` domain.
+
```

Comment:
I think we should remove all references to the legacy endpoints and their migrations (endpoints that use the realm domain are legacy endpoints)